pkg/apis/ovirtprovider/v1beta1/types.go

```diff
@@ -49,7 +49,23 @@ type OvirtMachineProviderSpec struct {
 	// the oVirt cluster this VM instance belongs too
 	ClusterId string `json:"cluster_id"`
 
+	// Sockets is the number of sockets for a VM.
+	// Total CPUs is Sockets * Cores * Threads
```

Comment:
Maybe surround with brackets (Sockets * Cores * Threads) to make it clearer it's a calculation

Reply:
ack